tests/wasm/0setup_server.go

```diff
@@ -0,0 +1,91 @@
+// +build ignore
```

Comment:
Is this right?

Reply:
It is intentional for the docker setup as it needed a separate server executable; but I'm going to remove docker support per the conversation with Ayke and simplify things.  The code from this file will move into a _test.go file and become directly part of the test setup/teardown.